charts/stable/paperless-ng/values.yaml

```diff
@@ -1,18 +1,51 @@
 image:
-  repository: tccr.io/truecharts/paperless-ng
+  repository: jonaswinkler/paperless-ng
   pullPolicy: IfNotPresent
-  tag: v1.5.0
+  tag: v1.5.0@sha256:e4102d1b850c9a54f7ac1f3a0be197f451ad36ae165b43aa9829714e1d582ca8
 
-securityContext:
-  runAsNonRoot: false
-  readOnlyRootFilesystem: false
+# securityContext:
+#   runAsNonRoot: false
+#   readOnlyRootFilesystem: false
 
-podSecurityContext:
-  runAsUser: 0
-  runAsGroup: 0
+# podSecurityContext:
+#   runAsUser: 0
+#   runAsGroup: 0
+
+secret:
+  PAPERLESS_ADMIN_USER: "admin"
+  PAPERLESS_ADMIN_PASSWORD: "admin"
+  PAPERLESS_ADMIN_MAIL: "admin@admin.com"
 
 env:
-  PUID: 568
+  USERMAP_UID: 568
+  USERMAP_GID: 568
+  PAPERLESS_TIME_ZONE: "{{ .Values.env.TZ }}"
+  # PAPERLESS_REDIS:
+  PAPERLESS_DATA_DIR: "/config/"
+  PAPERLESS_STATICDIR: "/static/"
```

Comment:
What has this to do with redis?
redis has no access to anything entered here... nor does paperless have access to datadirs inside the redis container.

Reply:
Nop, this was just the env for the redis url, which is commented out until is moved to envValueFrom (after making the changes to redisinjector